h2spatial-ext/src/main/java/org/h2gis/h2spatialext/function/spatial/properties/ST_Explode.java

```diff
@@ -222,4 +226,149 @@ public ResultSet getResultSet() throws SQLException {
             return rs;
         }
     }
```

Comment:
Yes. But I prefer  to have two different codes due to the aspect of the signature.